hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/s3/metrics/TestS3GatewayMetrics.java

```diff
@@ -110,4 +123,146 @@ public void testHeadObject() throws Exception {
     long curMetric = metrics.getHeadKeySuccess();
     assertEquals(1L, curMetric - oriMetric);
   }
-}
+
+  @Test
+  public void testGetBucketEndpointMetric() throws Exception {
```

Comment:
The reason for that is because these Unit tests are written for all the metrics associated with the Endpoints exposed by the S3 API! would you want me to remove it?

Reply:
Yes, please follow naming of existing test methods like `testHeadBucket`, `testListBucket`, `testHeadObject`.

Reply:
Sure will do!!